cmake/recipes/geogram.cmake

```diff
@@ -41,7 +41,7 @@ include(FetchContent)
 FetchContent_Declare(
     polyfem_geogram
 	GIT_REPOSITORY https://github.com/polyfem/geogram.git
-	GIT_TAG        516c151c244d9019a9076a1a468d52a0f6dd195d
+	GIT_TAG        c790c30
```

Comment:
What about using the [official release](https://github.com/BrunoLevy/geogram)? I tried it on my mac and it compiles...

Reply:
the proble was windows, no?

Reply:
yeah, but I cannot test on windows

Reply:
Windows is fixed. we can use tags or revert the change